src/BloomBrowserUI/bookEdit/css/bloomDialog.less

```diff
@@ -60,7 +60,7 @@
 .bloomDialogContainer BUTTON {
 	.bdRounded;
 	border: 1px solid #8abbf4;
-	box-shadow: #CCC 0 0 .25em;
+	box-shadow: 1px 2px 7px #888;
```

Comment:
updated this file to use the power of .less

might be more that could be done in pre-existing code